apps/test/unit/templates/lessonOverview/LessonOverviewTest.js

```diff
@@ -27,12 +27,14 @@ describe('LessonOverview', () => {
 
   it('renders default props', () => {
     const wrapper = shallow(<LessonOverview {...defaultProps} />);
-    expect(wrapper.contains('Lesson Name'));
-    expect(wrapper.contains('Lesson Overview'));
-    expect(
-      wrapper.contains('The purpose of the lesson is for people to learn')
+    expect(wrapper.contains('Lesson Name'), 'Lesson Name').to.be.true;
```

Comment:
while I was debugging, I used this syntax to give a more helpful error message when there were multiple `.to.be.true` statements in a row. however I realized after that something like this might be more helpful: `expect(wrapper.text()).to.contain('Foo')`

because we do not usually get accurate line numbers in unit test failures, it is a good idea to make sure there is enough info to identify the failure based on the test case name and the error message.